packages/overture-schema-places-theme/src/overture/schema/places/place.py

```diff
@@ -82,6 +82,50 @@ class Categories(BaseModel):
     ] = None
 
 
+@no_extra_fields
+class Taxonomy(BaseModel):
+    """
+    A structured representation of the place's category within the Overture taxonomy.
+
+    Provides the primary classification, full hierarchy path, and alternate categories.
+    """
+
+    # Required
+
+    primary: Annotated[
+        SnakeCaseString,
+        Field(
+            description="The most specific category for this place within the taxonomy hierarchy."
+        ),
+    ]
+
+    hierarchy: Annotated[
+        list[SnakeCaseString],
+        Field(
+            min_length=1,
+            description=textwrap.dedent("""
+                The full taxonomy path from the most general to most specific category,
+                represented as an ordered array (e.g., [food_and_drink, restaurant,
+                casual_eatery, gas_station_sushi]).
+            """).strip(),
+        ),
+    ]
+
+    # Optional
+
+    alternates: Annotated[
+        list[SnakeCaseString] | None,
+        Field(
+            description=textwrap.dedent("""
+                Additional categories that apply to this place but are not the primary
+                classification. Useful for places that fit multiple categories (e.g., a
+                gas station that also serves sushi).
```

Comment:
Is this a good example when the example for hierarchies already has `"gas_station_sushi"`?